Automattic/vip-go-mu-plugins
# AI-16: Add ability to shutdown site when overdue

## Description

Adds the ability to effectively shutdown a WordPress application, when the ~`WPCOM_VIP_OVERDUE_LOCKOUT`~ `VIP_OVERDUE_LOCKOUT` constant is set and truthy. 

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Check out PR.
2. Edit `config/wp-config.php` and add the following: 
```php
define( 'VIP_OVERDUE_LOCKOUT', true );
```
3. If testing on a VIP environment, you should see the `site-shutdown.html` page on all requests. 
4. If you are testing on a local environment, nothing should happen. Fake a VIP environment by adding 
```php
define('VIP_GO_ENV', true);
```
to your `wp-config.php` file
